docs/docsite/rst/user_guide/playbooks_reuse_roles.rst

```diff
@@ -163,7 +163,7 @@ You can conditionally import a role and execute it's tasks::
       tasks:
       - include_role:
           name: some_role
-        when: "ansible_os_family == 'RedHat'"
```

Comment:
Should this state `ansible_facts['os_platform']` like the above changes?

Reply:
no, the above are wrong

Reply:
@bcoca do you mean lines 49 and 51 should also be changed to `ansible_facts['os_family']`?